datastash/logstash.conf

```diff
@@ -0,0 +1,25 @@
+input {
+  elasticsearch {
+    hosts       => ""
+    index       => "*"
+    docinfo     => true
+    docinfo_target => "[@metadata][docinfo]"
+  }
+}
+
+filter {
+  mutate {
+    # Remove internal fields added by Logstash
+    remove_field => ["@version", "@timestamp"]
```

Comment:
out of curiousity, what does `@version` get set to?

Reply:
Seems like it's always set to `1`.

```
$ docker run --rm -it opensearchproject/logstash-oss-with-opensearch-output-plugin:7.13.4 -e 'input { stdin { } } output { stdout { } }'
.... 
Pipelines running {:count=>1, :running_pipelines=>[:main], :non_running_pipelines=>[]}
test
{
    "@timestamp" => 2023-04-13T00:52:12.444Z,
          "host" => "5140d3f222e1",
       "message" => "test",
      "@version" => "1"
}
```